data/liveinst/locale1-x11-sync.service

```diff
@@ -0,0 +1,7 @@
+[Unit]
+Description=Sync localed configuration with running X11 system
+
```

Comment:
Missing `[install]` section
https://docs.fedoraproject.org/en-US/packaging-guidelines/Systemd/#unit_files_basic_format_install

Reply:
Yeah, lets add that, pointing to the graphical target in our case.